docs/content/concepts/assets/asset-jobs.mdx

```diff
@@ -0,0 +1,194 @@
+---
+title: Asset jobs | Dagster
+description: Asset jobs are the main unit for materializing and monitoring Software-defined assets in Dagster.
+---
+
+# Asset jobs
+
+Jobs are the main unit for executing and monitoring Software-defined assets in Dagster. An asset job materializes a selection of [Software-defined Assets](/concepts/assets/software-defined-assets). It can be launched in a few different ways:
+
+- Manually from the Dagster UI
+- At fixed intervals, by [schedules](/concepts/partitions-schedules-sensors/schedules)
+- When external changes occur, using [sensors](/concepts/partitions-schedules-sensors/sensors)
+
+<!-- TODO -->
+
+<Note>
```

Comment:
I suggest we move this above the intro paragraph. That way it's the first thing someone sees when they come here, which is great if ops is what they're looking for.